example/notify.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env nix-shell
```

Comment:
We have a `contrib` directory in the repo already for "extra stuff around lorri that could be useful", like `notify.sh` and `prompt.sh`.

I think these scripts are really great in showing what users can do with this feature. Would you mind
- moving the scripts to `contrib`
- adding a tiny little guide in the style of the ones already present in `contrib` (see https://github.com/target/lorri/tree/master/contrib)?

Reply:
If you prefer, you can also remove `notify.sh` and `prompt.sh` from this PR and re-introduce them with docs in a follow-up PR. That could help keep the size of this PR down a bit. Whichever you think is easier.

Reply:
Done. That'll be first follow-up, since it also makes to elaborate the daemon.md with notify.sh